docs/getting_started/index.rst

```diff
@@ -146,6 +146,18 @@ to your local working copy):
 
    $ git pull upstream master
 
+--------------------------------------------
+Creating an environment for specific version
+--------------------------------------------
+
+A virtual environment for a specific version using the release on conda-forge can be created with::
+
+  CTAPIPE_VER=0.17.0
+  wget https://raw.githubusercontent.com/cta-observatory/ctapipe/v$CTAPIPE_VER/environment.yml
+  mamba env create -n cta-$CTAPIPE_VER -f environment.yml
+  mamba activate cta-0.17.0
```

Comment:
```suggestion
  mamba activate cta-$CTAPIPE_VER
```

Reply:
I'd remove this option completely. The `environment.yml` is for development (and then you just have it locally) and the CI (which is something we maybe don't even want in on env, since it e.g. doesn't make sense to install vitables in the ci).

People should install ctapipe directly, or add it to their env files. Not use the one from this repo.